lib/scala/project-manager/src/main/scala/org/enso/projectmanager/boot/ProjectManager.scala

```diff
@@ -278,14 +282,17 @@ object ProjectManager extends ZIOAppDefault with LazyLogging {
   private def displayVersion(
     useJson: Boolean
   ): ZIO[ZAny, IOException, ExitCode] = {
-    val versionDescription = VersionDescription.make(
+    val versionDescription = makeVersionDescription
```

Comment:
Local variable `versionDescription` can be removed and one can use ` makeVersionDescription.asString(useJson = false)` just like on line 234.